src/tests/ftest/soak/soak.py

```diff
@@ -434,6 +436,62 @@ def execute_soak_test(self, test_param, pools):
         if len(self.failed_job_id_list) > 0:
             raise SoakTestError("<<FAILED:  Soak {} >>".format(self.test_name))
 
+    def run_soak(self, test_param):
+        """Run the soak test specified by the test params.
+
+        Args:
+            test_param (str): test_params from yaml file
+
+        """
+        pool_list = self.params.get("poollist", test_param + "*")
+        self.test_timeout = self.params.get("test_timeout", test_param)
+        self.job_id_list = []
+        start_time = time.time()
+
+        rank = self.params.get("rank", "/run/container_reserved/*")
+        obj_class = self.params.get(
+            "object_class", "/run/container_reserved/*")
+
+        # Create the reserved pool with data
+        self.pool = self.create_pool(["pool_reserved"])
+        self.pool[0].connect()
+        container = TestContainer(self.pool[0])
```

Comment:
Change this variable to `self.container`:
```suggestion
        self.container = TestContainer(self.pool[0])
```